### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by TheOneAndOnlyKelly** [dlemstra/Magick.NET#656] [Help] [answered]
Is there anywhere where there is some documentation on how to translate the ImageMagick commands that someone's already written into the .Net version? I have this from a website:

convert thumbnail.gif -alpha set -virtual-pixel transparent -channel A -blur 0x8 -level 50%,100% +channel  soft_edge.png

that I would like to convert. It's a command that puts a nice soft edge on the original picture.

**Comment by TheOneAndOnlyKelly** [comment, marked as answer]
I figured out how to convert the command.

**Reply by dlemstra** [reply]
For future readers of this message. Most ImageMagick options are methods or properties on the `MagickImage` class.

p.s. It would be nice if you could include the code in the message that you posted @TheOneAndOnlyKelly.
